x-pack/plugins/observability/server/saved_objects/slo.ts

```diff
@@ -56,6 +56,7 @@ export const slo: SavedObjectsType = {
       },
       revision: { type: 'short' },
       enabled: { type: 'boolean' },
+      tags: { type: 'keyword' },
```

Comment:
I'm concerned that tags are part of the SO's model... Are we referring to the same SO Tagging tags that already exist in Kibana?

If so, I'll defer the recommendations to the owners: @elastic/appex-sharedux

Reply:
@afharo I did not found the documentation for saved object own tags. Do you have it?

Reply:
Here is what I found: https://github.com/elastic/kibana/tree/main/x-pack/plugins/saved_objects_tagging

Reply:
I believe this feature is only accessible from the UI, e.g. allow customers to tag a SO from the saved object management page. I have not found a way to tag a SO when creating/updating the SO from the API. 
For the SLO project, we are using SO to store definition, but we don't expect/want the customer to tag the SO from the Kibana tagging system.

Am I missing something @afharo @simianhacker